keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -366,17 +366,17 @@ kind: BucketAccessClass
 metadata: 
   name:
 provisioner: [1]
-policyActions: [2]
-  allow:
-  - "*"
-  deny:
-  - "*"
-parameters: [3]
+policyActionsConfigMap: [2]
+  - name: [3]
+    namespace: [4]
+parameters: [5]
 ```
 
 1. `provisioner`: (required) the name of the driver that `BucketAccess` instances should be managed by. Format: <provisioner-namespace>"/"<provisioner-name>, eg "ceph-rgw-provisoning/ceph-rgw.cosi.ceph.com".
-1. `policyActions`: a set of provisioner/platform defined policy actions to allow or deny a given user identity.
-1. `parameters`: (Optional)  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values.
+1. `policyActionsConfigMap`: (required) a ConfigMap that contains a set of provisioner/platform defined policy actions tos a given user identity.
+1. `name`: (required) name for the *policyActionsConfigMap*.
+1. `namespace`: (required) namespace of the *policyActionsConfigMap*.
+1. `parameters`: (Optional)  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Optional reserved keys cosi.io/configMap and cosi.io/secrets are used to reference user created resources with provider specific access policies.
```

Comment:
"Optional reserved keys cosi.io/configMap and cosi.io/secrets are used to reference user created resources with provider specific access policies."

Is ^ this true for MVP? I feet it is may not be required for MVP.